### PR TITLE
style: Change color scheme to blue/white (#18)

### LINKED ISSUE #18: Propose some color schemes and themes for the templates.
Currently we dont have uniform colors throughout the app. Please propose some color schemes.

### PR DESCRIPTION
**Pull request by mrmir**
Fixes #18.

Deployment link: https://mrmir1.herokuapp.com/

### CODE REVIEW

File: public/css/styles.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .chat-body {
-  background-color: #B5CC18;
+  background-color: white;
```
Comment (sumedh123) [inline public/css/styles.css]: use hex values

File: public/css/styles.css [hunk 2]
```diff
@@ -77,7 +77,7 @@
   right: 5px;
   font-size: 0.7em;
   bottom: 5px;
-  color: black;
+  color: white;
```
Comment (sumedh123) [inline public/css/styles.css]: here too

**Comment by sumedh123** [comment]
Also make the color of timestamp white.

**Comment by mrmir** [comment]
@sumedh123 Please review!

**Pull request merged**